packages/components/src/components/input/input.stories.ts

```diff
@@ -69,731 +35,219 @@ export const Default = {
+      <sd-input size="sm" label="Label"> </sd-input>
+    </div>
+  `
 };
 
 /**
- * Add descriptive help text to an input with the `help-text` attribute. For help texts that contain HTML, use the `help-text` slot instead.
+ * Use the `label` attribute to give the input an accessible label.
+ *
+ * For labels that contain HTML, use the `label` slot instead.
  */
 
-export const HelpText = {
-  name: 'Help Text',
-  args: overrideArgs([
-    { type: 'attribute', name: 'label', value: 'Nickname' },
-    { type: 'attribute', name: 'help-text', value: 'How would you like to be called?' }
-  ]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args
-        })}
-      </div>
-    `;
-  }
+export const Label = {
+  render: () => html`
+    <div class="flex flex-row gap-12 items-baseline">
+      <sd-input label="Label" spellcheck></sd-input>
+      <sd-input spellcheck>
+        <div slot="label" class="text-lg">Label</div>
+      </sd-input>
+    </div>
+  `
 };
 
 /**
- * Use the `placeholder` attribute to add a placeholder.
+ *Use the `placeholder` attribute to add a placeholder.
  */
-
-export const Placeholders = {
-  args: overrideArgs([{ type: 'attribute', name: 'placeholder', value: 'Type something' }]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args
-        })}
-      </div>
-    `;
-  }
-};
-
-/**
- * Add the `clearable` attribute to add a clear button when the input has content.
- */
-
-export const Clearable = {
-  args: overrideArgs([{ type: 'attribute', name: 'clearable', value: true }]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args
-        })}
-      </div>
-    `;
-  }
+export const Placeholder = {
+  render: () => html`
+    <div class="w-[250px]">
+      <sd-input placeholder="Placeholder example" spellcheck></sd-input>
+    </div>
+  `
 };
 
 /**
- * Add the `password-toggle` attribute to add a toggle button that will show the password when activated. Only works with `type="password"`.
+ * Use the `disabled` attribute to disable the input.
  */
 
-export const TogglePassword = {
-  name: 'Toggle Password',
-  parameters: {
-    controls: {
-      exclude: ['password-toggle']
-    }
-  },
-  args: overrideArgs([{ type: 'attribute', name: 'type', value: 'password' }]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args,
-          constants: [
-            { type: 'attribute', name: 'password-toggle', value: true },
-            { type: 'attribute', name: 'label', value: 'Password' }
-          ]
-        })}
-      </div>
-    `;
-  }
+export const Disabled = {
+  render: () =>
+    html`<div class="w-[250px]">
+      <sd-input label="Label" placeholder="Input text disabled" disabled></sd-input>
```

Comment:
We should also add a "cursor-not-allowed" to the disabled input

Reply:
https://github.com/orgs/solid-design-system/projects/1/views/1?pane=issue&itemId=78382912